chips/msp432/src/flctl.rs

```diff
@@ -0,0 +1,849 @@
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        BMRK OFFSET(8) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        PRG_ERR OFFSET(9) NUMBITS(1) []
+    ],
+    FLCTL_SETIFG [
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        RDBRST OFFSET(0) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        AVPRE OFFSET(1) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        AVPST OFFSET(2) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        PRG OFFSET(3) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        PRGB OFFSET(4) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        ERASE OFFSET(5) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        BMRK OFFSET(8) NUMBITS(1) [],
+        /// Write 1 clears the corresponding interrupt flag bit in the FLCTL_IFG
+        PRG_ERR OFFSET(9) NUMBITS(1) []
+    ],
+    FLCTL_READ_TIMCTL [
+        /// Configures the length of the Setup phase for this operation
+        SETUP OFFSET(0) NUMBITS(8) [],
+        /// Length of the IREF_BOOST1 signal of the IP
+        IREF_BOOST1 OFFSET(12) NUMBITS(4) [],
+        /// Length of the Setup time into read mode when the device is recovering from one o
+        SETUP_LONG OFFSET(16) NUMBITS(8) []
+    ],
+    FLCTL_PRGVER_TIMCTL [
+        /// Length of the Setup phase for this operation
+        SETUP OFFSET(0) NUMBITS(8) [],
+        /// Length of the Active phase for this operation
+        ACTIVE OFFSET(8) NUMBITS(4) [],
+        /// Length of the Hold phase for this operation
+        HOLD OFFSET(12) NUMBITS(4) []
+    ],
+    FLCTL_PROGRAM_TIMCTL [
+        /// Length of the Setup phase for this operation
+        SETUP OFFSET(0) NUMBITS(8) [],
+        /// Length of the Active phase for this operation
+        ACTIVE OFFSET(8) NUMBITS(20) [],
+        /// Length of the Hold phase for this operation
+        HOLD OFFSET(28) NUMBITS(4) []
+    ],
+    FLCTL_ERASE_TIMCTL [
+        /// Length of the Setup phase for this operation
+        SETUP OFFSET(0) NUMBITS(8) [],
+        /// Length of the Active phase for this operation
+        ACTIVE OFFSET(8) NUMBITS(20) [],
+        /// Length of the Hold phase for this operation
+        HOLD OFFSET(28) NUMBITS(4) []
+    ],
+    FLCTL_MASSERASE_TIMCTL [
+        /// Length of the time for which LDO Boost Signal is kept active
+        BOOST_ACTIVE OFFSET(0) NUMBITS(8) [],
+        /// Length for which Flash deactivates the LDO Boost signal before processing any ne
+        BOOST_HOLD OFFSET(8) NUMBITS(8) []
+    ]
+];
+
+#[repr(u32)]
+#[derive(Copy, Clone)]
+pub enum WaitStates {
```

Comment:
`///` comment here (what are the `WaitStates` used for?)

Reply:
Ok!